docs/variants/protectli_vp46xx/firmware-update.md

```diff
@@ -9,10 +9,11 @@ firmware version.
 For simplicity we recommend using
 [Dasharo Tools Suite](../../../common-coreboot-docs/dasharo_tools_suite).
 
-## Updating to Dasharo v1.0.18
+## Updating to Dasharo v1.0.18 or newer versions
```

Comment:
if one updates from v1.0.18 to v1.0.19 flashing only WP_RO and RW_SECTION_A is sufficient. BUt if v1.0.19 is goign to be the first Dasahro version being installed, full flashing is needed